core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -173,8 +173,6 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
                     webexp.routes()
                 } ~ {
                     swaggerRoutes
-                } ~ options {
```

Comment:
If we leave it were it is currently, it will never match the OPTIONS route for web actions.